owncloudData/src/test/java/com/owncloud/android/data/files/datasources/implementation/OCLocalFileDataSourceTest.kt

```diff
@@ -112,6 +112,17 @@ class OCLocalFileDataSourceTest {
         verify(exactly = 1) { fileDao.getFileByIdAsFlow(OC_FILE_ENTITY.id) }
     }
 
+    @Test
+    fun `getFilesForAccount returns a list of OCFile`() {
+        every { fileDao.getFilesForAccount(OC_ACCOUNT_NAME) } returns listOf(OC_FILE_ENTITY)
+
+        val result = ocLocalFileDataSource.getFilesForAccount(OC_ACCOUNT_NAME)
+
+        assertEquals(listOf(OC_FILE), result)
+
+        verify(exactly = 1) { fileDao.getFilesForAccount(OC_ACCOUNT_NAME) }
+    }
```

Comment:
You have to add a test to see what happens when DAO returns an empty list. It's similar to `getFolderContent` method and its tests, didn't you see?

Reply:
Yeah I was thinking to add it but in the function we don't have a condition to handle if the list is empty. I didn't find it very useful. Btw I added it, the rest of the tests have this implementation as well.

Reply:
We're giving a treatment for the result, in this case `toModel()`, we need to check this is working fine as well for empty lists